src/vm/eventtrace.cpp

```diff
@@ -1774,6 +1790,12 @@ void BulkTypeEventLogger::FireBulkTypeEvent()
     m_nBulkTypeValueByteCount = 0;
 }
 
+#else
+void BulkTypeEventLogger::FireBulkTypeEvent()
+{
+//   _ASSERTE(!"Eventing Not Implemented");
```

Comment:
Please remove the commented out code. If its purpose was to indicate that we will need to implement this in the future, please use `// UNIXTODO: xxxx` instead.
.